src/Symfony/Component/HttpKernel/ControllerMetadata/Argument/ArgumentMetadata.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\ControllerMetadata\Argument;
```

Comment:
But these classes only make sense in the context of the `Controller` subnamespace, so having them in a non-related namespace seems weird to me.